matlab/lib/+Ice/EndpointInfo.m

```diff
@@ -61,26 +47,33 @@
             if ~isempty(obj.underlying)
                 r = obj.underlying.secure();
             else
-                r = obj.secure_;
+                r = false;
             end
         end
     end
-    properties(SetAccess=private)
-        % underlying   The information of the underyling endpoint or an empty
-        %   array if there's no underlying endpoint.
+    properties(SetAccess=immutable)
+        % underlying   The information of the underlying endpoint or an empty array if there's no underlying endpoint.
```

Comment:
For a follow up PR:

If you look at the example doc comments from https://www.mathworks.com/help/matlab/matlab_prog/create-help-for-classes.html 

They don't have the property name at the start.